src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,20 +145,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
-
+    // allows users to enter basic HTML tags that are not a threat to security
+    String HTMLMessageContent = clean(messageContent, Whitelist.basic());
```

Comment:
Since you're also initializing the basic whitelist on line 167, you don't need to pass it into the method here (and can remove it from the type signature. That way, the detail of what whitelist is used stays in the part of the code that needs it (the clean() method).